### ISSUE TITLE
Error while trying to scan a sonarqube image

### ISSUE THREAD

**Issue by donchkat** [anchore/grype#1105]
**What happened**:

I tried to scan a sonarqube image version 9.5.0-community and got the next error

```
/home/ubuntu# grype sonarqube:9.5.0-community
 ✔ Vulnerability DB        [updated]
New version of grype is available: 0.56.0 (currently running: 0.54.0)
 ✔ Loaded image            
 ✔ Parsed image            
 ⠏ Cataloging packages     [packages 0]
1 error occurred:
	* failed to catalog: 1 error occurred:
	* failed to resolve files by glob (**/java): cycle during symlink resolution
```

**What you expected to happen**:

I expected to get the scan results.

**How to reproduce it (as minimally and precisely as possible)**:

pull the image:

```docker pull sonarqube:9.5.0-community```

Scan it:
```grype sonarqube:9.5.0-community```

**Anything else we need to know?**:

I managed to scan the same image with other scanners like Trivy, so the image is okay.
Also, I managed to scan the latest version of sonarqube image with Grype.

**Environment**:
- Output of `grype version`: 0.54.0
- OS (e.g: `cat /etc/os-release` or similar):

```PRETTY_NAME="Ubuntu 22.04.1 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.1 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
```

**Comment by westonsteimel** [comment]
I believe this should already be resolved in the latest release

**Comment by tgerla** [comment]
We've confirmed that this problem is fixed in v0.56.0 so I'll close this issue. Let us know if you need anything else!
